ProjectSource/build.gradle

```diff
@@ -109,5 +126,10 @@ dependencies {
     testCompile 'org.powermock:powermock-module-junit4:1.6.6'
     testCompile 'org.powermock:powermock-api-mockito:1.6.6'
     testCompile "org.mockito:mockito-core:1.10.8"
+
+    testCompile 'org.assertj:assertj-core:2.8.0'
```

Comment:
⚠️  Dependency version is specified as literal in this Gradle build file. An improvement would be to use `gradle.properties` for defining dependency versions and using variables when specifying them in configurations.  

**[More details](https://confluence.devfactory.com/x/9g2UEw)**
          
## The possible solution in this case

gradle.properties:
```Groovy
assertj_core_version = 2.8.0
```
build.gradle:
```Groovy
testCompile "org.assertj:assertj-core:$assertj_core_version"
```
 

<a id="0a0000d0-50c2-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />